build/images/scripts/start_ovs_netdev

```diff
@@ -25,7 +25,10 @@ function add_br_phy {
     log_info $CONTAINER_NAME "Creating OVS br-phy bridge for netdev datapath type"
     hwaddr=$(ip link show eth0 | grep link/ether | awk '{print $2}')
     inet=$(ip addr show eth0 | grep "inet " | awk '{ print $2 }')
+    inet6=$(ip addr show eth0 | grep "inet6 " | grep global | awk '{ print $2 }')
     gw=$(ip route | grep default | awk '{ print $3 }')
+    gw6=$(ip -6 route | grep default | awk '{ print $3 }')
+    ipv6enabled=$(sysctl net.ipv6.conf.all.disable_ipv6 --values)
```

Comment:
Can you add a comment to explain that eth0 may be assigned an IPv6 interface even when IPv6 is disabled which is why we check the value of the sysctl kernel parameter independently?